JSDemos/Demos/Scheduler/SimpleArray/description.md

```diff
@@ -1 +1,5 @@
-In&nbsp;this demo, the Scheduler component visualizes data stored in&nbsp;an&nbsp;in-memory array. The component is&nbsp;bound to&nbsp;this array using the **dataSource** property. Note that object structure complies with the Default Item Structure, that&nbsp;is, each object has the **text**, **startDate** and **endDate** fields.
\ No newline at end of file
+You can use the Scheduler component to display and edit appointments from a local array. Use the [dataSource](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#dataSource) property to bind the Scheduler to the array.
+
+The Scheduler automatically displays appointments if their data source fields match the fields described in the [dxSchedulerAppointment](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/) help section. In this demo, the [text](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#text), [startDate](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#startDate), [endDate](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#endDate), and [allDay](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#allDay) fields illustrate the case. If field names in your data source differ, specify them in **...Expr** properties ([textExpr](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#textExpr), [startDateExpr](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#startDateExpr), [endDateExpr](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#endDateExpr), [allDayExpr](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#allDayExpr), and so on).
```

Comment:
Two notes here: 

- You use "appointment" and "data source object" interchangeably (as synonyms), which seems a bit confusing: "displays appointments if **their** data source fields ...". I'd rather make a clear separation between the notion of "appointment" (as something that the scheduler displays) and "data source object". In other words: "if data source object contains required fields, an appointment appears". But those are not fields of the appointment. 
- I would move the list of property names to the end of the sentence, so I can first complete the thought, and only then list all the settings related to that. Whenever you have a long modifier or a list of some kind, it's best to avoid having it in the middle of the sentence. 

```suggestion
The Scheduler automatically displays appointments if source object field names match the names listed in the [dxSchedulerAppointment](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/) help section. In this demo, the following fields illustrate this point: [text](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#text), [startDate](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#startDate), [endDate](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#endDate), and [allDay](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#allDay). If field names in your data source differ, specify them in **...Expr** properties ([textExpr](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#textExpr), [startDateExpr](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#startDateExpr), [endDateExpr](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#endDateExpr), [allDayExpr](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#allDayExpr), and so on).
```